Sources/SPTPersistentCache.m

```diff
@@ -910,14 +903,12 @@ - (void)dispatchEmptyResponseWithResult:(SPTPersistentCacheResponseCode)result
                                callback:(SPTDataCacheResponseCallback)callback
                                 onQueue:(dispatch_queue_t)queue
 {
-    if (callback != nil) {
+    [self dispatchBlock:^ {
         SPTPersistentCacheResponse *response = [[SPTPersistentCacheResponse alloc] initWithResult:result
                                                                                             error:nil
                                                                                            record:nil];
-        dispatch_async(queue, ^{
-            callback(response);
-        });
-    }
+        callback(response);
```

Comment:
The static analyzer picks up when we attempt to call nil on a block without checking, and all our blocks have been fully qualified in this method. There isn't in the error dispatch method which is why we need the check there